src/main/java/com/truelayer/java/mandates/entities/mandatedetail/MandateDetail.java

```diff
@@ -30,7 +30,7 @@ public abstract class MandateDetail {
 
     private String reference;
 
-    private User user;
```

Comment:
Is it just a change on the Get payment response or also on the mandate detail ? In the last case, was this propagated to the API ref and relevant code in the RecPay stack ?